apps/admin-portal/src/components/users/add-user-groups.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { ReactElement, useState } from "react";
+import _ from "lodash";
+import { Forms } from "@wso2is/forms";
+import { TransferComponent, TransferList, TransferListItem } from "@wso2is/react-components";
+
+/**
+ * Proptypes for the application consents list component.
+ */
+interface AddUserGroupProps {
+    initialValues: any;
+    triggerSubmit: boolean;
+    onSubmit: (values: any) => void;
+    handleGroupListChange: (groups: any) => void;
+    handleTempListChange: (groups: any) => void;
+}
+
+/**
+ * User role component.
+ *
+ * @return {ReactElement}
+ */
+export const AddUserGroup: React.FunctionComponent<AddUserGroupProps> = (props: AddUserGroupProps): ReactElement => {
```

Comment:
Add the `FunctionComponent` to the already existing react import.

```suggestion
export const AddUserGroup: FunctionComponent<AddUserGroupProps> = (props: AddUserGroupProps): ReactElement => {
```